### ISSUE TITLE
Depth prediction training loop

### ISSUE THREAD

**Issue by justinwaltrip** [justinwaltrip/FCRN-colorization#3]
- [ ] loss curves
- [ ] add task layer changes
- [ ] update hyperparams
